docs/atlascli/command/atlas-streams-instances-update.txt

```diff
@@ -0,0 +1,110 @@
+.. _atlas-streams-instances-update:
+
+==============================
+atlas streams instances update
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Updates an Atlas Stream Processing instance for your project.
+
+Before updating an Atlas Streams Processing instance, you must first stop all processes associated with it.
+To use this command, you must authenticate with a user account or an API key that has the Project Owner role.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas streams instances update <name> [options]
+
+.. Code end marker, please don't delete this comment
+
+Arguments
+---------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - name
+     - string
+     - true
+     - Name of the Atlas Stream Processing instance. After creation, you can't change the name of the instance. The name can contain ASCII letters, numbers, and hyphens.
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for update
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file. To see full output, use the -o json option.
+   * - --projectId
+     - string
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+   * - --provider
+     - string
+     - true
+     - Cloud service provider that applies to the provisioned Atlas Stream Processing instance. This value defaults to "AWS".
+   * - -r, --region
+     - string
+     - true
+     - Human-readable label that identifies the physical location of your Atlas Stream Processing instance. The region can affect network latency and performance if it is far from your source or sink.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.
+
+Output
+------
+
+If the command succeeds, the CLI returns output similar to the following sample. Values in brackets represent your values.
+
+.. code-block::
+
+   Atlas Streams Processor Instance '<Name>' successfully updated.
+   
+
+Examples
+--------
+
+.. code-block::
+
+   # Modify the Atlas Stream Processing instance configuration with the ID 5d1113b25a115342cca2d1aa:
+   atlas streams instances update 5d1113b25a115342cca2d1aa --provider AWS --provider VIRGINIA_USA
```

Comment:
atlas streams instances update 64f89335caf8bc352465a8e6 --provider AWS --region VIRGINIA_USA

Reply:
This should be SPI name and not ID

Reply:
Resolved in [82c2186](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/82c2186c49ac36d4c34bfb1057ac3be1bb66941d)